woocommerce/woocommerce
# add lerna hoist, use consistent package versions

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds lerna hoist and updates package versions across all package.json files to use the same versions.

Closes # .

### How to test the changes in this Pull Request:

1. Run `npm run install`.
2. Ensure E2E tests run.

### Other information:


### Changelog entry

N/A
